resources/user.rb

```diff
@@ -19,10 +19,10 @@
     # Check if we need to change the password.
     if self.class.properties[:password].is_set?(desired)
       begin
-        ::Nexus3::Api.new(api_client.endpoint, username, desired.password).request(:get, '/service/metrics/ping')
-        password 'Supercalifragilisticexpialidocious-that-does-not-exist-so-maybe-the-resource-will-need-to-converge'
-      rescue ::Nexus3::ApiError
+        ::Nexus3::Api.new(api_client.endpoint, username, desired.password).request(:get, '/service/rapture/session')
```

Comment:
As discussed on Slack the `/service/wonderland/authenticate` endpoint requires User tokens feature to be enabled on the Nexus instance, which might not always be the case. I updated to instead use the basic status endpoint - I tried locally and with wrong credentials this will return 401, which should serve our use case.